extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -20,7 +20,7 @@ class UserPreferencesV2 {
 	 * @return Bool
 	 */
 	static public function onGetPreferences( $user, &$defaultPreferences ) {
-		global $wgEnableWallExt, $wgEnableDiscussions, $wgOut, $wgScriptPath, $wgServer, $wgUser, $wgAuth;
+		global $wgEnableWallExt, $wgOut, $wgScriptPath, $wgServer, $wgUser, $wgAuth;
 
 		// add javascript
 		// TODO: use $wgExtensionsPath instead
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)